app/models/agency.py

```diff
@@ -31,6 +31,15 @@ class Agency(db.Model):
     incident_reports = db.relationship('IncidentReport', backref='agency',
                                        lazy='joined')
 
+    def __init__(self, **kwargs):
+        super(Agency, self).__init__(**kwargs)
```

Comment:
hmm what's Agency's super? how come we do this and not `super(**kwargs)`? if this is standard for python pls ignore, but otherwise add a comment?